CHANGELOG.next.toml

```diff
@@ -1,46 +1,5 @@
-# Example changelog entries
-# [[aws-sdk-rust]]
-# message = "Fix typos in module documentation for generated crates"
-# references = ["smithy-rs#920"]
-# meta = { "breaking" = false, "tada" = false, "bug" = false }
-# author = "rcoh"
-#
-# [[smithy-rs]]
-# message = "Fix typos in module documentation for generated crates"
-# references = ["smithy-rs#920"]
-# meta = { "breaking" = false, "tada" = false, "bug" = false }
-# author = "rcoh"
```

Comment:
something went wrong merging the changelog. Can you leave the example changelog entries there as well?

Reply:
I have put the examples back